doc/en/user/source/data/app-schema/joining.rst

```diff
@@ -83,3 +83,14 @@ Much like joining support, native encoding of nested filters is turned on by def
 Or, alternatively, by setting the value of the Java System Property "app-schema.encodeNestedFilters" to "false", for example ::
 
      java -DGEOSERVER_DATA_DIR=... -Dapp-schema.encodeNestedFilters=false Start
+
+UNION performance improvement for OR conditions
+-----------------------------------------------
+
+Is known that OR queries are difficult to optimize for postgresql and are usually slow.  App-Schema improve OR condition performance using UNION clauses instead OR for nested filter subqueries.
+
+This improvement is enabled by default but there is a System Property that allows you to enable/disable it: "app-schema.orUnionReplace".  For example if you want to disable the UNION improvements you should use::
+
+     app-schema.orUnionReplace = false
+	
+This performance improvement is isolated for Postgresql datasources.
```

Comment:
This deserves to be a note \ warning, and I would reword it: ``This optimization will only be applied when a PostgresSQL database is being used.``.